src/components/events/BannerImage.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import Image from "next/image";
+import classroom from "@/public/events/classroom.png";
+
+const BannerImage = () => {
+  return (
+    <>
+      <div>
+        <Image
+          src={classroom}
+          className="top-[97.8px] w-screen"
```

Comment:
```suggestion
          className="aspect-[x/y] w-screen"
```

find appropriate x and y values

Reply:
Why do we use these properties instead of setting the top of the image?